disk_test.go

```diff
@@ -0,0 +1,282 @@
+package skewer
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2022-08-01/compute" //nolint:staticcheck
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+func Test_SKU_HasSCSISupport(t *testing.T) {
+	cases := map[string]struct {
+		sku    compute.ResourceSku
+		expect bool
+	}{
+		"empty capability list should return true (backward compatibility)": {
+			sku:    compute.ResourceSku{},
+			expect: true,
+		},
+		"no disk controller capability should return true": {
+			sku: compute.ResourceSku{
+				Capabilities: &[]compute.ResourceSkuCapabilities{
+					{
+						Name:  to.StringPtr("vCPUs"),
+						Value: to.StringPtr("8"),
+					},
```

Comment:
Why not just an empty list? (here and elsewhere)

Reply:
Fixed